src/ClientHandle.cpp

```diff
@@ -1117,17 +1117,15 @@ void cClientHandle::HandleLeftClick(Vector3i a_BlockPos, eBlockFace a_BlockFace,
 		/* Check for clickthrough-blocks:
 		When the user breaks a fire block, the client send the wrong block location.
 		We must find the right block with the face direction. */
-
-		Vector3i BlockPos = a_BlockPos;
-		AddFaceDirection(BlockPos, a_BlockFace);
```

Comment:
Can we kill the original AddFaceDirection that takes a reference now? Would be good if we just had one function with one semantic rather than two that do different things with no warning.